examples/experimental/dagster-airlift/dagster_airlift/core/serialization/compute.py

```diff
@@ -23,29 +25,73 @@
 from dagster_airlift.core.utils import spec_iterator
 
 
+@record
+class TaskSpecMappingInfo:
```

Comment:
prefer language to not be specific to tasks since we're retrieving information that goes beyond just the spec mapping info

Reply:
@dpeng817 I'm going to merge this and do that rename at top of stack to avoid rebasing challenges

Reply:
https://app.graphite.dev/github/pr/dagster-io/dagster/24714/s-TaskSpecMappingInfo-AirliftMetadataMappingInfo